config/envoyconfig/envoyconfig.go

```diff
@@ -92,6 +93,13 @@ func buildAccessLogs(options *config.Options) []*envoy_config_accesslog_v3.Acces
 		return nil
 	}
 
+	var additionalRequestHeaders []string
+	for _, field := range options.AccessLogFields {
+		if headerName, ok := log.GetHeaderField(field); ok {
+			additionalRequestHeaders = append(additionalRequestHeaders, http.CanonicalHeaderKey(headerName))
```

Comment:
Do we intend to support HTTP/2 pseudo-headers (`:authority`, `:method`, etc.)? I think CanonicalHeaderKey won't canonicalize those (cf. https://go.dev/play/p/MRbeUn11-fp).

(Does that matter here? I'm not sure.)

Reply:
Yes they are supported. `headers.:authority` worked for me. Should we write custom code to lower case `:` header names?

Reply:
I could go either way. It's probably fine not to convert something like `headers.:Authority`; that feels like an odd thing to type.